homeassistant/components/recorder/__init__.py

```diff
@@ -96,6 +98,8 @@
                     vol.Coerce(int), vol.Range(min=0)
                 ),
                 vol.Optional(CONF_DB_URL): cv.string,
+                vol.Optional(CONF_DB_MAX_RETRIES): cv.positive_int,
+                vol.Optional(CONF_DB_RETRY_WAIT): cv.positive_int,
```

Comment:
Use the declared constants DEFAULT_DB_MAX_RETRIES, DEFAULT_DB_RETRY_WAIT as default values in the config schema. See the [official documentation](http://alecthomas.github.io/voluptuous/docs/_build/html/voluptuous.html?highlight=default%20value#voluptuous.schema_builder.Optional) for reference.
```suggestion
                vol.Optional(CONF_DB_RETRY_WAIT, default=DEFAULT_DB_RETRY_WAIT): cv.positive_int,
```

Reply:
Implemented suggested change in 8bac36bdd014bd9f2f5299aae40019bf23ea6ad7